src/AutomationManagement/Microsoft.WindowsAzure.Management.Automation.nuspec

```diff
@@ -23,7 +23,7 @@
       </group>
     </references>
     <dependencies>
-      <dependency id="Microsoft.Azure.Common" version="[2.0.3,3.0)" />
+      <dependency id="Microsoft.Azure.Common" version="[2.0.2,3.0)" />
```

Comment:
This actually won't work - the msi installer in PowerSehll will not allow a downgrade